modules/certificate/certificate.service.js

```diff
@@ -2,22 +2,29 @@ const Certificate = require('./certificate.model');
 const RuleError = require('../../errors/rule.error');
 
 const {
-  STATUS_CODES: { NOT_FOUND, BAD_REQUEST },
+  STATUS_CODES: { NOT_FOUND, BAD_REQUEST, UNAUTHORIZED },
 } = require('../../consts/status-codes');
 const {
   CERTIFICATE_NOT_FOUND,
-  CERTIFICATE_ALREADY_EXISTS,
+  CERTIFICATE_NOT_VALID,
+  CERTIFICATE_HAVE_OWNER,
+  CERTIFICATE_IS_ACTIVE,
+  CERTIFICATE_IS_USED,
 } = require('../../error-messages/certificate.messages');
+const { USER_NOT_AUTHORIZED } = require('../../error-messages/user.messages');
 
 class CertificatesService {
   async getAllCertificates(skip, limit, user) {
     let filter;
 
     if (user.role === 'admin' || user.role === 'superadmin') {
```

Comment:
why do you need this? set let filter = {}

Reply:
Nice comment, will fix it. Shame for me I didn't think of it.

Reply:
done